python/cuml/dask/common/utils.py

```diff
@@ -133,7 +135,13 @@ def persist_across_workers(client, objects, workers=None):
     """
     if workers is None:
         workers = client.has_what().keys()  # Default to all workers
-    return client.persist(objects, workers={o: workers for o in objects})
+
+    if check_min_dask_version("2020.12.0"):
```

Comment:
I'm guessing this is the release when the new annotation feature was added? Have you been able to test on this release?

Reply:
Yes, I tested on that release successfully.